src/stylesheets/project_specific/developer/_forms.scss

```diff
@@ -1,5 +1,85 @@
 
-  /* Braintree-specific class */
-  .field_with_errors {
-    @extend .has-error;
-  }
+/* Braintree-specific class */
+.field_with_errors {
+  @extend .has-error;
+}
+
+.dollah-sign {
+  display: inline-block;
+  float: left;
+  margin-right: 6px;
+  font-size: 24px;
+  line-height: 40px; // to match input field height
+}
+
+input[disabled],
+input[disabled] + .dollah-sign {
+  color: #ccc;
+}
+
+// Toggle Switch
+// Should we move to common/_forms partial?
+$toggle-height: 16px;
+$toggle-width: 40px;
+$toggle-transition-time: 400ms;
+$toggle-disabled-color: #777777;
+$toggle-active-color: #00A37A;
+
+.toggle-switch {
+  position: relative;
+  display: inline-block;
+  width: $toggle-width;
+  height: $toggle-height;
+}
+
+// Hide default HTML checkbox
+.toggle-switch input {display:none;}
+
+.toggle-switch .slider {
```

Comment:
Do we use the `.slider` class anywhere else? would this name ever be confused with the range slider? (I would name `.slider` `.toggle-switch-slider` to make it clear what component this is attached to.)